example.md

```diff
@@ -0,0 +1,590 @@
+
+[![Build Status](https://travis-ci.org/protofire/eth-cli.svg?branch=master)](https://travis-ci.org/protofire/eth-cli)
+[![NPM version](https://badge.fury.io/js/eth-cli.svg)](https://npmjs.org/package/eth-cli)
+[![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/protofire/eth-cli/master/LICENSE)
+[![dependencies Status](https://david-dm.org/protofire/eth-cli/status.svg)](https://david-dm.org/protofire/eth-cli)
+[![devDependencies Status](https://david-dm.org/protofire/eth-cli/dev-status.svg)](https://david-dm.org/protofire/eth-cli?type=dev)
+
+CLI swiss knife for Ethereum developers.
+
+## What is this
+
+`eth-cli` is a collection of commands that can help you develop dapps
+for Ethereum. Check the [Examples](#examples)
+section or the [list of commands](#commands) to find out what you can do with it.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [eth-cli](#eth-cli)
+  - [What is this](#what-is-this)
+  - [Examples](#examples)
+    - [Start a REPL to interact with an RPC node](#start-a-repl-to-interact-with-an-rpc-node)
+    - [Generate a random address](#generate-a-random-address)
+    - [Get info about the most popular chains](#get-info-about-the-most-popular-chains)
+  - [Installation](#installation)
+  - [Commands](#commands)
+    - [`abi`](#abi)
+      - [`abi:list`](#abilist)
+      - [`abi:methods`](#abimethods)
+      - [`abi:show`](#abishow)
+    - [`conf`](#conf)
+      - [`conf:abi:add`](#confabiadd)
+      - [`conf:abi:remove`](#confabiremove)
+      - [`conf:address:add`](#confaddressadd)
+      - [`conf:address:get`](#confaddressget)
+      - [`conf:address:list`](#confaddresslist)
+      - [`conf:address:remove`](#confaddressremove)
+    - [`contract`](#contract)
+      - [`contract:address`](#contractaddress)
+      - [`contract:deploy`](#contractdeploy)
+    - [`method`](#method)
+      - [`method:decode`](#methoddecode)
+      - [`method:encode`](#methodencode)
+      - [`method:hash`](#methodhash)
+      - [`method:send`](#methodsend)
+      - [`method:send-transaction`](#methodsend-transaction)
+    - [`networks`](#networks)
+    - [`randomAddress`](#randomaddress)
+    - [`transaction`](#transaction)
+      - [`transaction:get`](#transactionget)
+      - [`transaction:nop`](#transactionnop)
+    - [`repl`](#repl)
+  - [Back us](#back-us)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+
+## Examples
+
+There are a lot of things that you can do with `eth-cli`, and we keep adding more. These are some of our favorites:
+
+
+### Use help to learn how to play around with eth-cli
+
+```
```

Comment:
@janus How do you get this output?

Reply:
Please , which output are you referring?